Automattic/vip-go-mu-plugins
# Wp-admin: Add notice for Search customers without the constant

## Description
Since March 1 will be here before we know it, it would be good to display a notice.
<img width="1718" alt="Screenshot 2023-01-25 at 3 11 28 PM" src="https://user-images.githubusercontent.com/16962021/214704036-c9e9fcf6-8265-41ff-8bf0-33254c7c2085.png">


## Changelog Description

### Updated: Wp-admin

Add reminder to test out Enterprise Search upgrade 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Define search constant:
```
define( 'VIP_ENABLE_VIP_SEARCH', true ); // Enables Enterprise Search
```
2) Hit up wp-admin with super admin
3) Expect to see notice
4) Add testing constant:
```
define( 'VIP_SEARCH_USE_NEXT_EP', true );
```
5) Repeat step 2 
6) Expect to see notice gone
